Engine/Client/Src/Application/Application.cpp

```diff
@@ -52,17 +56,69 @@ void Application::_keyboardHandler(std::size_t key)
         // if (!_sceneManager->processInput(KEY_MAP(key), this->_client->getIdxPlayerComponent()))

```

Comment:
Delete it if it's not useful anymore

Reply:
For the moment, we can't run this while the game is not running on the client/room.